health/health.d/x509check.conf

```
@@ -9,7 +9,9 @@ component: x509 certificates
     every: 60s
      warn: $this < $days_until_expiration_warning*24*60*60
      crit: $this < $days_until_expiration_critical*24*60*60
-     info: time until x509 certificate expires
+     info: Time until x509 certificate expires. \
+           X509 certificate will expire soon. \
```

```suggestion
     info: Time until the X.509 certificate expires. \
           The X.509 certificate will expire soon. \
```

`X.509` is the correct formatting here, as it’s part of the ITU X.509 standard.